sig-autoscaling/charter.md

```diff
@@ -0,0 +1,78 @@
+# SIG Autoscaling Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+Covers development and maintenance of components for automated scaling in
+Kubernetes.  This includes automated vertical and horizontal pod
+autoscaling, initial resource estimation, cluster-proportional system
+component autoscaling, and autoscaling of Kubernetes clusters themselves.
+
+### In scope
+
+- Autoscaling-related API objects, such as the HorizontalPodAutoscaler and
+  VerticalPodAutoscaler
+
+- Autoscaling-related tools, such as the cluster autoscaler,
+  single-component scaling tools (e.g. pod-nanny), and
+  cluster-proportional scaling tools
+
+- Ensuring API interfaces (the scale subresource) are available and usable
+  to enable other SIG to write autoscalable objects, and enable people to
+  interact with those interfaces.
+
+[Link to SIG section in sigs.yaml][sigs.yaml]
+
+#### Code, Binaries and Services
+
+- Components and utilities that take automated action to scale a component
+  on the cluster
+
+- Components and utilities that take automated action to scale the cluster
+  itself
+
+- Special parts of client-go for interacting with with the scaling
+  interfaces used by the HPA (e.g. the polymorphic scale client).
+
+#### Cross-cutting and Externally Facing Processes
+
+- Reviewing implementations of the scale subresource to ensure that
+  autoscaling behaves properly
+
+- Coordinating with SIG Instrumentation to ensure that metrics APIs are
+  suitable for autoscaling on.
+
+- Coordinating with SIG Scheduling to make sure scheduling decisions can
+  interact well with the cluster autoscaler
+
```

Comment:
Possibly something about cluster lifecycle wrt to cluster autoscale?

Reply:
also SIG node for vertical scaling needs in the kubelet?

Reply:
yep, those are both correct.  I'll add them.